docs/msbuild/walkthrough-creating-an-msbuild-project-file-from-scratch.md

```diff
@@ -99,7 +101,7 @@ If you're running on Windows, you can use the **Developer Command Prompt for Vis
 
 ### To create a minimal MSBuild project file
 
-1. In the text editor, create a new file *HelloWorld.csproj* and enter the following code:
+1. In the text editor, create a new file *HelloWorld.fromscratchproj* and enter the following code:
```

Comment:
@ghogen do you think we should explicitly call out that both filename and extension are arbitrary, though ending in `proj` is conventional? Not sure if that's too much to explain in this context.

Reply:
@rainersigwald 
I think what you say should be mentioned in the top-level MSBuild overview, rather than here. I think for our purposes here, we could use the extension ".proj" or as you did, ".fromscratchproj".

Reply:
Internal PR created. https://github.com/MicrosoftDocs/visualstudio-docs-pr/pull/12284